.prettierrc

```diff
@@ -1,3 +1,4 @@
 {
-  "singleQuote": true
-}
\ No newline at end of file
+  "singleQuote": true,
+  "trailingCommas": "es5"
```

Comment:
@lucleray Is this sufficient or does something need to be changed in eslint config?

Reply:
This is perfect 👍 Eslint rules that overlap with prettier are deactivated by `eslint-config-prettier` here:
https://github.com/zeit/now/blob/cc0bd9f0a226232c4919c14183c73b8138c6c238/.eslintrc.json#L13